frontend/src/util/events.js

```diff
@@ -0,0 +1,3 @@
+export const dispatchEvent = (name, detail) => {
```

Comment:
It's a little risky giving this the same name as a function in `document` right? I guess it's not a huge issue but it could cause some confusion.

Reply:
yeah, I might look into using `dispatchCustomEvent` or something